.obs/chartfile/operator/values.yaml

```diff
@@ -10,8 +10,9 @@ seedImage:
   imagePullPolicy: IfNotPresent
 
 channel:
```

Comment:
I am also wondering if this shouldn't be a list of channels. Makes sense from a structure point of view, but I wonder if this wouldn't make it way cumbersome and unpleasant from a UX PoV.

Reply:
I would also opt to not introduce many channels at install time. One default is great, then we can point to the documentation to install additional ones. Otherwise it will be a lot of clutter.